bok_choy/page_object.py

```diff
@@ -338,6 +338,20 @@ def wait_for_page(self, timeout=30):
         Raises:
             BrokenPromise: The timeout is exceeded without the page loading successfully.
         """
+        def _is_document_ready():
+            """
+            Check the loading state of the document to ensure the document
+            and all sub-resources have finished loading.
+            """
+            return self.browser.execute_script(
+                "return document.readyState=='complete'")
+
+        EmptyPromise(
+            _is_document_ready,
+            "The document load event has been fired.",
+            timeout=timeout
```

Comment:
Not a must, but could this say something more similar to the docstring in _is_document_ready, like `the document and all sub-resources have finished loading`?